web/src/lib/components/utilities-page/duplicates/duplicates-compare-control.svelte

```diff
@@ -27,6 +27,15 @@
   let selectedAssetIds = $state(new SvelteSet<string>());
   let trashCount = $derived(assets.length - selectedAssetIds.size);
 
+  const uniqueDateTimes = new Set(
+    assets.map((asset) => {
+      const date = new Date(asset.fileCreatedAt);
```

Comment:
The display correctly uses dateTimeOriginal/localDateTime, but this uses fileCreatedAt - why the difference?